firebase_admin/mlkit.py

```diff
@@ -27,6 +27,7 @@
 
 
 _MLKIT_ATTRIBUTE = '_mlkit'
+MAX_PAGE_SIZE = 100
```

Comment:
Prefix with `_` to indicate that it's not part of the public API.

Reply:
done